site/app/views/forum/ForumThreadView.php

```diff
@@ -847,7 +848,54 @@ public function createThread() {
 		 </script>
 
 		<div style="margin-top:5px;background-color:transparent; margin: !important auto;padding:0px;box-shadow: none;" class="content">
+HTML;
+		if($this->core->getUser()->getGroup() <= 2){
+			$categories = $this->core->getQueries()->getCategories();
+			$return .= <<<HTML
+			<div class="popup-form" id="category-list">
+				<h3>Categories</h3>
+				<pre>(Drag to re-order)</pre><br>
+HTML;
+				if(count($categories) == 0) {
+					$return .= <<<HTML
+					<span id='category-list-no-element' style="margin-left: 1em;" >
+						No category found
```

Comment:
If there is no category found there still displays a draggable bar. Can we please remove that? Also if there exists no categories we should instruct the user to create one. So possibly something like "No categories exist please create one." or something similar. 


![draggablebar](https://user-images.githubusercontent.com/16356240/41197190-16fba3c6-6c23-11e8-817a-192f378c25e1.png)

Reply:
Thanks! Bug fixed.